### PR TITLE
adding FAQ page

### PR DESCRIPTION
**Pull request by crazy4pi314**
Trying to answer some common questions!

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+If you are looking for a team, please check out the `#find-a-team` channel on the [Discord](http://discord.unitary.fund) or post on your favorite social media with the hashtag `#unitaryhack`.
+
+## Q. How difficult is this hackathon?
+
+All skill levels can participate in unitaryHACK. We have worked with maintainers to make sure there are issues available for all levels of experience, both in quantum as well as open source contributions.
+If you are having trouble finding issues you want to work on, be sure to ask on the [Discord](http://discord.unitary.fund) and the organizing team can brainstorm with you.
+
+## Q. What are qualifying pull requests?
+
+Hackers can make contributions anytime between May 14th and May 30th. Just be sure to [sign up here](https://7r2lxl70pvl.typeform.com/to/HfJLX5fl) and make sure that you put `[unitaryHACK]` in the title of your pull request so the bots can find your contributions and make sure you get credit!
```
Comment (nathanshammah) [inline faq.md]:
```suggestion
Hackers can make contributions anytime between May 14th and May 30th. Just be sure to [sign up here](https://airtable.com/embed/shrTG20PLC5cjVIPr) and make sure that you put `[unitaryHACK]` in the title of your pull request so the bots can find your contributions and make sure you get credit!
```

File: faq.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+## Q. Will there be mentors who can help us during the event?
+
+Totally! You can find our friendly, open source quantum community on our [Discord](http://discord.unitary.fund), where you can as questions in chat and at office hours as well.
+Office hours schedule will be posted on Discord closer to the event.
+
+## Q. What technology is the focus of this event? (commercial app)
+
+Any quantum open source project listed on our [participating projects](./participating-projects.md) page.
+You will find project built with a wide variety of languages like Python, Q#, Julia, Ruby, C++, and more.
```
Comment (nathanshammah) [inline faq.md]:
```suggestion
You will find projects built with a wide variety of languages like Python, Q#, Julia, Ruby, C++, and more.
```

File: faq.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+There is a good mix between industry and applications focused projects as well as some that are more on the research side.
```
Comment (nathanshammah) [inline faq.md]:
```suggestion
There is a good mix between industry-focused and application-focused projects as well as some that are more on the research side.
```

Comment (nathanshammah) [reply faq.md]: (I think)

File: faq.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+We have tons of good resources for you over on the [hacker guide](./hacker-guide.md)!
+
+## Q. Is there specific quantum computing knowledge required?
+
+Nope! You can start learning the day the event starts if you want!
+We have lots of great resources on the [hacker guide](./hacker-guide.md) if you want to get started early though.
+
+## Q. What timezone is it in?
+
+All times for the events will be in [PDT](https://time.is/PDT).
```
Comment (nathanshammah) [inline faq.md]: I wonder if the timezone point is really needed. To me it sounds a bit like discouraging for folks in other timezones, whereas the main message here I guess is that:
- times for deadlines, etc., are mainly PDT (but this can simply be added when needed)
- some interactive meetings, Q&As (Twitch, Discord, etc.) will be hosted at times suitable for folks in PDT. However, being the hack fully asynchronous, it relatively affects contributors. 

My vote is to omit it and just use PDT when expressing times.

File: faq.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+Absolutely!
+You are free to work on teams, just make sure that everyone's username is mentioned in the pull request!
+Please note that the bounties are awarded per solution/pull request and it is up to the team to split the money.
+
+If you are looking for a team, please check out the `#find-a-team` channel on the [Discord](http://discord.unitary.fund) or post on your favorite social media with the hashtag `#unitaryhack`.
+
+## Q. How difficult is this hackathon?
+
+All skill levels can participate in unitaryHACK. We have worked with maintainers to make sure there are issues available for all levels of experience, both in quantum as well as open source contributions.
```
Comment (nathanshammah) [inline faq.md]:
```suggestion
Contributors with any skill level can participate in unitaryHACK. We have worked with maintainers to make sure there are issues available for all levels of experience, both for quantum-related as well as simply open-source-related contributions.
```

File: faq.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+The hackathon will run from May 14-30th, and you can register at any point before the event is over.
+
+## Q. Does it cost anything?
+
+Nope, but YOU can get paid if you work on an issue with a bounty and your contribution is selected by the project maintainers!
+See the list of [participating projects](./participating-projects.md) to see the bountied issues for each repo.
+
+## Q. How does the hackathon work?
+
+**tl;dr**: Make a contribution to a [participating open source project](./participating-projects.md) between May 14-30th and you will win SWAG. Contribute a solution to a bountied issue and we will send you CASH via PayPal.
```
Comment (nathanshammah) [inline faq.md]:
```suggestion
**tl;dr**: Make an approved contribution to a [participating open source project](./participating-projects.md) between May 14-30th and you will win SWAG. If your contribution to a bountied issue is deemed the satisfactory solution by the maintainers of the project, we will send you cash via PayPal.
```

Comment (nathanshammah) [reply faq.md]: Wanted to clarify this bit, as not simply opening a PR would qualify.

File: faq.md (hunk 1 above)
Comment (nathanshammah) [reply faq.md]: Switched from the Typeform the Airtable link, which is the recently used one when one also clicks on "sign up" from the home page.

**Review by nathanshammah — CHANGES_REQUESTED** [review]
Thanks @crazy4pi314, added some suggestions and comments.

**Comment by crazy4pi314** [comment]
Thanks for the feedback @nathanshammah 💖 Will push live now for marketing tomorrow
